share/man/man9/cdefs.9

```diff
@@ -0,0 +1,297 @@
+.\"-
+.\" Copyright (c) 2024 M. Warner Losh <imp@FreeBSD.org>
+.\"
+.\" SPDX-License-Identifier: BSD-2-Clause
+.\"
+.Dd July 3, 2024
+.Dt CDEFS 9
+.Os
+.Sh NAME
+.Nm cdefs
+.Nd "Compiler portability definitions"
+.Sh DESCRIPTION
+.In sys/cdefs.h
+defines macros for compiler, C language standard portability, POSIX standards
+compliance and symbol visibility.
+It defines programming interfaces for the system header files to adopt to the
+many environments
+.Fx
+supports compilation for.
+It defines convenience macros for the
+.Fx
+sources, tailored to the base
+system's needs.
```

Comment:
```suggestion
sources, tailored to the requirements of the base operating system.
```

Avoiding an apostrophe.

Reply:
I don't like this. This is massively more verbose and we say base system in other manuals.

Also, you can use Ap in mdoc.
For example, the example Ap s grammar.

Reply:
I don't avoid apostrophes. Sorry.
It's not a contraction, but the possessive form.
So no. Just no.